authorization.md

```diff
@@ -24,20 +32,62 @@ Each call has to to include HTTP header generated and signed by CreatorsHub:
 
 Sample request may look as follows:
 
-```
+```sh
 curl --location --request POST '{INSTANCE_URL}/api/public/batchMint' \
---header 'Authorization: Bearer {JWT}' \
---header 'Content-Type: application/json' \
---data-raw '[
-   {
-       "title": "Bear 1",
-       "status": "draft",
-       "amount": 1
-   },
-]'
+  --header 'Authorization: Bearer {JWT}' \
+  --header 'Content-Type: application/json' \
+  --data '[
+      {
+        "title": "Bear 1",
+        "status": "draft",
+        "amount": 1
+      }
+    ]'
 ```
 
-## Obtaining the API Token
+### Obtaining the API Token
 
 The endpoint is secured by the API Token attached to the User who generated the token.
 In order to generate the token log in to our Instance, go to `Settings`, find `API Access` under `Settings` list, go to the `Marketplace API` tab and click the `Generate token` button or grab the Token if you already generated one.
+
+## User cookie
+
+Our public API also works when you're logged in on some instance without need of passing the `Authorization` header along with valid API Token.
+
+!!!warning Warning
+It is recommended to access API using [access token generated by admin](#token). Retrieving info from public instance endpoints being authenticated using user cookie shouldn't be permanent solution if you want to build something on top of the API :rocket:
+!!!
+
+Sample request may look as follows:
+
+```sh
+curl --location --request POST '{INSTANCE_URL}/api/public/batchMint' \
+  --header 'Content-Type: application/json' \
+  --header 'Cookie: token={COOKIE}; HttpOnly; Secure' \
+  --data '[
+      {
+        "title": "Bear 1",
+        "status": "draft",
+        "amount": 1
+      }
+    ]'
+```
+
+### Obtaining the cookie
```

Comment:
I'd add information that obtaining the cookie only makes sense for debug/develop/test purposes. Otherwise, the browser should automatically deliver the cookie whenever an HTTP call is made.